src/tools/emacs.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Emacs
+description: Use Dart on Emacs and its flavors
+toc: false
+---
+
+Emacs has a few major and minor modes built for Dart and Flutter development.
+
+All the steps are identical for enabling Flutter since
+[Dart-Snippets][dart-snippets] contain Flutter snippets too.
+[Flutter.el][flutter-mode] only allows you to run Flutter commands inside your Emacs.
+
+* [Dart-Mode][dart-mode] is the main mode for development.
+* [Dart-Snippets][dart-snippets] enables [Visual Studio Code](/tools/vs-code)
+  like snippets and more.
+* [LSP-Mode][lsp-mode] adds IDE-like features to Dart mode.
+
+{% comment %}
+
+Add a vanilla Emacs config that has all above + hooks and stuff.
+
+{% endcomment %}
+  
+To enable and configure all above in [Doom Emacs][doom-emacs], you simply
```

Comment:
Not clear what "doom emacs" is; the rest of this page is just talking about Emacs.

Reply:
(having read up on doom emacs, I would recommend we not try to involve it here.)

Reply:
I'd argue... It's the single most used Emacs configuration framework probably (not sure tho).
And I personally put this up for my classmates to get Dart installed on their Doom. Who happen to use Doom, all of them.

Unfortunately, I am not experienced enough. If I was I'd surely instruct installing the modules on other "flavours" of Emacs too (like Spacemacs).

As the description suggests. It's about "Emacs and its flavours" not just the vanilla.

It has a lot of room to debate... Thanks addressing again. But I personally haven't seen a single person who uses the vanilla.